packages/ethereum-contracts/contracts/interfaces/utils/IUserDefinedMacro.sol

```diff
@@ -31,23 +31,16 @@ interface IUserDefinedMacro {
     function postCheck(ISuperfluid host, bytes memory params, address msgSender) external view;
 
     /*
-     * Additional to the required interface, we recommend to implement the following function:
-     * `function getParams(...) external view returns (bytes memory);`
+     * Additional to the required interface, we recommend to implement one or multiple view functions
```

Comment:
So, we avoid breaking change to user defined macro interface. That should be okay.